### PR TITLE
Fix sync_otp page translation

### PR DESCRIPTION
**Pull request by stanislavlevin**
Easy fix for:
https://pagure.io/freeipa/issue/7621
Depends on #2072

### CODE REVIEW

**Comment by abbra** [comment]
This pull request does depend on PR#2072, right?

**Comment by stanislavlevin** [comment]
Yes, it does (at translation part).

**Comment by abbra** [comment]
So, we first need to get 2072 merged first. I talked to @pvoborni  today and he promised to run a review ASAP.

**Comment by stanislavlevin** [comment]
Thank you so much!

File: install/ui/src/freeipa/plugins/sync_otp.js [hunk 1]
```diff
@@ -26,10 +26,12 @@ define(['dojo/_base/declare',
         '../navigation',
         '../phases',
         '../reg',
+	'../text',
```
Comment (pvoborni) [inline install/ui/src/freeipa/plugins/sync_otp.js]: Mixed tabs and spaces.

File: install/ui/src/freeipa/widgets/SyncOTPScreen.js [hunk 2]
```diff
@@ -27,14 +27,16 @@ define(['dojo/_base/declare',
         'dojo/on',
         '../ipa',
         '../auth',
+	'../config',
```
Comment (pvoborni) [inline install/ui/src/freeipa/widgets/SyncOTPScreen.js]: Mixed tabs and spaces.

**Comment by stanislavlevin** [comment]
I coped :)

**Comment by stanislavlevin** [comment]
Ping.

**Comment by Tiboris** [comment]
@serg-cymbaluk can you have a look?

**Comment by serg-cymbaluk** [comment]
Looks good to me.

**Comment by serg-cymbaluk** [comment]
master:

* adfd82ee74e9bfb221d77fda64a3e77d72f7f5ac Replace the direct URL with config's one
* 1721356d536c696450143f94863fb4e7386d98cf Fix translation of "sync_otp" plugin
* 14e21047e4c80369dafe584887fb073fb7dcf906 Fix translation of "SyncOTPScreen" widget

**Pull request closed without merging**